src/tests/multihost/alltests/test_backtrace.py

```diff
@@ -17,6 +17,11 @@
 from constants import ds_instance_name
 
 
+def check_login(user, hostname, password):
```

Comment:
Can the check_login function be moved into a common library since it's defined in so many places and then just import it?